docs/api.md

```diff
@@ -51,13 +51,13 @@ An array of functions to be called after Electron has been extracted to a tempor
 
 When `true`, sets both [`arch`](#arch) and [`platform`](#platform) to `all`.
 
-##### `app-copyright`
+##### `appCopyright`
```

Comment:
Please keep the options alphabetized, so probably place the deprecated options somewhere above their respective new names.